base/install_opcua.sh

```diff
@@ -5,13 +5,19 @@ set -ex
 . /opt/epics/install-functions.sh
 
 opcua_release_url=https://github.com/epics-modules/opcua/releases/download/v${OPCUA_VERSION}
-opcua_release_file=IOC_opcua-${OPCUA_VERSION}_Base-${EPICS_BASE_VERSION}_debian${DEBIAN_VERSION%.*}.tar.gz
+opcua_release_file=BDIST_opcua-${OPCUA_VERSION}_Base-${EPICS_BASE_VERSION}_debian${DEBIAN_VERSION%.*}.tar.gz
 lnls-get-n-unpack -l $opcua_release_url/$opcua_release_file
 
-mv binaryOpcuaIoc opcua
-install_module -i opcua OPCUA "
+mv opcuaBinaryDist opcua-module
+install_module opcua-module OPCUA "
 EPICS_BASE
 "
 
+mv opcuaExampleIoc opcua
+install_module -i opcua OPCUA_IOC "
+EPICS_BASE
+OPCUA
+"
```

Comment:
As discussed in person, mention that `opcua/` is kept for backward compatibility with existing iocsh scripts.